### ISSUE TITLE
Step Counter Value 100 is shown as `...`

### ISSUE THREAD

**Issue by besi** [xmartlabs/XLForm#429]
When I step from 99 to 100, then `100` is not shown but `...`

![screenshot](https://cloud.githubusercontent.com/assets/160412/8701080/088fb0c2-2b10-11e5-8eda-43627a921ffc.png)

This is my config:

```ruby

row = XLFormRowDescriptor(tag: "rate", rowType: XLFormRowDescriptorTypeStepCounter, title: "Rate (%)")
row.cellConfigAtConfigure["stepControl.wraps"] = true
row.cellConfigAtConfigure["stepControl.value"] = 30
row.cellConfigAtConfigure["stepControl.stepValue"] = 10
row.cellConfigAtConfigure["stepControl.maximumValue"] = 100
row.cellConfigAtConfigure["stepControl.minimumValue"] = 10
row.cellConfigAtConfigure["stepControl.maximumValue"] = 100
```

**Comment by tommypeps** [comment]
First,
[stepControl object --> UIStepper Class] (https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIStepper_Class/#//apple_ref/occ/instp/UIStepper/maximumValue)

Maximun value by default 100.

You  have to must changer:
```objective-C
row.cellConfigAtConfigure["stepControl.maximumValue"] = 2000 
```

The value change but doesn't show on UILabel, I'm waiting to update Xcode and I will send  a pull request @besi @mtnbarreto

**Comment by besi** [comment]
I actually need 100 as the maximum value in my case, since I want to customise a percentage.

**Comment by pastorin** [comment]
@besi

This issue was fixed in the pull request #448.
Thanks for reporting it.

Regards,
Martin
